easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.9.0-foss-2023b.eb

```diff
@@ -0,0 +1,38 @@
+# #
```

Comment:
Not a big deal, but this looks a bit weird :) Older EasyConfigs had `##` without space. I'm also fine with removing line 1 alltogether and replace 3 with an empty line.